index.js

```diff
@@ -374,30 +375,73 @@ Pbf.prototype = {
     }
 };
 
-function readVarintRemainder(val, pbf) {
-    var buf = pbf.buf, b;
+function readVarintRemainder(l, u, p) {
```

Comment:
This function uses single character variables names to keep it under 600 characters. Longer functions are no inlined by v8, which drastically decreases performance.